### PR TITLE
fix(MessageAttachment): fixed display peer info title

### PR DESCRIPTION
**Pull request by pvlv**

### CODE REVIEW

File: src/components/MessageAttachment/MessageAttachment.css [hunk 1]
```diff
@@ -38,7 +38,8 @@
 }
 
 .from {
-  height: var(--message-attachment-header-height);
+  height: 100%;
```
Comment (meafmira) [inline src/components/MessageAttachment/MessageAttachment.css]: Why not change variable here?

File: src/components/MessageAttachment/MessageAttachment.css [hunk 2]
```diff
@@ -53,11 +54,10 @@
 }
 
 .fromButton {
-  padding: 0;
-  min-width: 0;
+  padding: 0 !important;
```
Comment (meafmira) [inline src/components/MessageAttachment/MessageAttachment.css]: Why use !important so frequently? Refactor this if needed

File: src/components/PeerInfoTitle/PeerInfoTitle.js [hunk 3]
```diff
@@ -24,11 +24,12 @@ type Props = {
   emojiSize?: number,
   isVerified?: ?boolean,
   isFluid?: ?boolean,
+  wrap?: ?boolean,
```
Comment (meafmira) [inline src/components/PeerInfoTitle/PeerInfoTitle.js]: Use `wrap: boolean` type here with default `false` value. Do not use optionals for boolean type.

File: src/components/MessageAttachment/MessageAttachment.css (hunk 1 above)
Comment (pvlv) [reply src/components/MessageAttachment/MessageAttachment.css]: height == 100% not used anywhere

**Pull request merged**